notebooks/6 Dictionaries.ipynb

```diff
@@ -53,24 +45,24 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Referencing keys in a dictionary gives you their values."
+    "Just like regular variables, you can overwrite values in dictionaries by assigning a different value to the key."
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "print( shakespeare_dict['first_name'] + ' ' + shakespeare_dict['last_name'] )\n",
-    "## This prints 'William Shakespeare'"
+    "To access an individual value in the dictionary, you need to provide the key that is associated with this value in square brackets appended to the name of the dictionary. "
```

Comment:
I think you usually know a *key* and then find out what value is associated with it. Here it sounds like you already know a *value* that you want to look up. Or am I missing something?